tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -132,6 +178,13 @@ def test_get_metrics_and_expectations(
         "data_asset_name": "my_reports",
     }
 
+    validator: Validator = get_validator_with_temporary_expectation_suite(
```

Comment:
I know it's a small detail (and only for tests), but I really like the clarity of this util function name :) thank you for this